h/api/models/token.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+import uuid
+
+import sqlalchemy
+
+from h.api import db
+
+
+class Token(db.Base):
+
+    """A long-lived API token for a user."""
+
+    __tablename__ = 'token'
+    userid = sqlalchemy.Column(sqlalchemy.UnicodeText(),
+                               primary_key=True)
```

Comment:
I think it would make more sense for the token, possibly called `id`, to be the PK. There might be good reasons in the future for a user to have multiple types of token associated with their account, but there is never going to be a good reason to have multiple tokens with the same id.